tests/unit/amo/components/TestAddonAdminLinks.js

```diff
@@ -0,0 +1,219 @@
+import * as React from 'react';
+
+import AddonAdminLinks, {
+  AddonAdminLinksBase,
+} from 'amo/components/AddonAdminLinks';
+import {
+  ADDONS_CONTENTREVIEW,
+  ADDONS_EDIT,
+  ADDONS_POSTREVIEW,
+  ADMIN_TOOLS_VIEW,
+  THEMES_REVIEW,
+} from 'core/constants';
+import { createInternalAddon } from 'core/reducers/addons';
+import {
+  dispatchClientMetadata,
+  dispatchSignInActions,
+  fakeAddon,
+  fakeTheme,
+} from 'tests/unit/amo/helpers';
+import { fakeI18n, shallowUntilTarget } from 'tests/unit/helpers';
+
+describe(__filename, () => {
+  let store;
+  const slug = 'some-slug';
+
+  beforeEach(() => {
+    store = dispatchClientMetadata().store;
+  });
+
+  function render(props = {}) {
+    return shallowUntilTarget(
+      <AddonAdminLinks
+        addon={props.addon || createInternalAddon(fakeAddon)}
+        i18n={fakeI18n()}
+        store={store}
+        {...props}
+      />,
+      AddonAdminLinksBase,
+    );
+  }
+
+  const renderWithPermissions = ({ addon, permissions }) => {
+    const perms = Array.isArray(permissions) ? permissions : [permissions];
+    return render({
+      addon:
+        addon === undefined
+          ? createInternalAddon({
+              ...fakeAddon,
+              slug,
+            })
+          : addon,
+      store: dispatchSignInActions({
```

Comment:
If you wanted to, you could just re-use the local store:

```js
const renderWithPermissions = ({ addon, permissions }) => {
  perms = ...;
  dispatchSignInActions({ store, userProps: { permissions: perms } });
  return render({ ... });
};
```

Without that, it might be surprising if someone tried to dispatch something from the local store.